themes/geekboot/assets/scss/docs.scss

```diff
@@ -50,4 +50,7 @@
 /* purgecss end ignore */
 @import "skippy";
 @import "fonts";
-@import "mermaid";
\ No newline at end of file
+// TODO(negz): Does import order matter? Can this go in the existing ignore block above?
```

Comment:
Mermaid might eventually depend on fonts so it would need to come after that. I don't know what skippy is.